lib/tasks/local_dev.rake

```diff
@@ -1,15 +1,30 @@
 desc 'Set up your local development environment with data from Find'
 task setup_local_dev_data: %i[environment copy_feature_flags_from_production sync_dev_providers_and_open_courses] do
   puts 'Creating a provider-only user with DfE Sign-in UID `dev-provider` and email `provider@example.com`...'
-  ProviderUser.find_or_create_by!(dfe_sign_in_uid: 'dev-provider', email_address: 'provider@example.com') do |u|
+  ProviderUser.find_or_create_by!(
+    dfe_sign_in_uid: 'dev-provider',
+    email_address: 'provider@example.com',
+    first_name: 'Apply',
+    last_name: 'Provider',
```

Comment:
As discussed, a real-sounding name would avoid confusion in the UI. 

Peter Rovider and Susan Upport?